tests/gold_tests/pluginTest/cert_update/ssl/client1.crt

```diff
@@ -0,0 +1,31 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I noticed that only the .pem files are used in the autests.  Can we drop the .crt and .key files, or add autests to test loading them as separate files vs as combined into the .pem file?

Reply:
You are right. I should remove .crt and .key files since the plugin is assuming they are in the same .pem file.